index.html

```diff
@@ -5,34 +5,55 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="style.css">
+    <title>My portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600&display=swap" rel="stylesheet">
 </head>
 
 <body>
 
-    <div class="first-block first-block--color first-block--size">
-        <span class="first-block__first-element"></span>
-    </div>
-
-    <div class="second-block second-block--color second-block--size">
-        <span class="second-block__first-element"></span>
-        <span class="second-block__second-element"></span>
-    </div>
+    <div class="container">
```

Comment:
K HTML mám jenom jednu připomínku, ale důležitou - vytvořila sis tu takovou malou říši divů. :joy: Sémantických elementů, mezi které `<div>` patří, ale existuje mnohem víc, viz [zde](https://www.w3schools.com/html/html5_semantic_elements.asp), minimálně hlavní container by měl být `<main>`, jeho obsah by měly být `<section>` nebo `<article>` a pak až divy. Footer má taky vlastní element `<footer>`. Je to primárně kvůli čitelnosti, ale i tak důležitý znak toho, že rozumíš tomu, co děláš. :sunny:

Reply:
Já jsem byla strašně zmatená po tom prvním úkolu a vůbec jsem netušila co s projektem dál. Chtěla jsem to vzdát, ptž jsem po BEM vůbec nevěděla co s tím dál, pochopila jsem to tak, že už ty elementy, co popisuješ, nemohu používat.

Stále jsem z toho upřímně hodně zmatená a nejraději bych to celé od začátku přepsala, ale už nebyl čas.